### ISSUE TITLE
why my wav2lip_train.py is always keep sync_loss = 0.0

### ISSUE THREAD

**Issue by DiaosWang** [primepake/wav2lip_288x288#44]
the command is :
$ python wav2lip_train.py --data_root enhanced/ --checkpoint_dir checkpoints/  --syncnet_checkpoint_path 
checkpoints/syncnet_checkpoint_step000120000.pth --checkpoint_path checkpoints/checkpoint_step000075000.pth

the result of train is:
L1: 0.02834209179919627, Sync Loss: 0.0: : 540it [09:59,  1.05s/it]

why the Sync loss is always =0.0 ?

**Comment by SixpenniesS** [comment]
bro,you should read the code in wav2lip_train.py. Probably at line 228 of the code

**Comment by primepake** [comment]
sync_loss in the first step, the second you need to set the weight of it
